infrastructure/lib/kit-infrastructure.ts

```diff
@@ -139,11 +139,26 @@ export class KITInfrastructure extends Stack {
                     "eks:*",
                     "pricing:GetProducts",
                     "sts:AssumeRole",
-                    "s3:*"
+                    "s3:*",
+                    "sqs:*",
+                    "fis:*"
                 ],
             }),
         ],
     }));
+    // Add FIS permissions for EC2 interruptions
+    sa.role.addManagedPolicy(iam.ManagedPolicy.fromAwsManagedPolicyName('service-role/AWSFaultInjectionSimulatorEC2Access'))
+    sa.role.grantAssumeRole(new iam.ServicePrincipal("fis.amazonaws.com", 
+    {
+      conditions: {
+        StringEquals: {
+          "aws:SourceAccount": Stack.of(this).account,
+        },
+        ArnLike: {
+          "aws:SourceArn": `arn:${Stack.of(this).partition}:fis:${Stack.of(this).region}:${Stack.of(this).account}:experiment/*`
```

Comment:
Why does it say `experiment` ? Is this not GA ?

Reply:
experiments is an FIS resource owned by the FIS service. It has nothing to do with the lifecycle of the resource.